python/res/enkf/jobs/enkf_scaling_get_data.py

```diff
@@ -0,0 +1,173 @@
+from copy import deepcopy
+
+import pandas as pd
+
+from res.enkf.export import GenDataCollector, SummaryCollector, SummaryObservationCollector
+
+
+def get_measured_data(ert, events):
+    """
+    Main entry point, will read simulation and observed data,
+    filter on user specified index, and filter out outliers.
+    """
+
+    measured_data = _get_data(ert, events.keys)
+
+    measured_data = measured_data.dropna(axis=1)
+
+    measured_data = _filter_on_column_index(measured_data, events.index)
+
+    measured_data = _filter_out_outliers(measured_data, events)
+
+    if measured_data.shape[1] < 2:
+        raise ValueError(
+            "Not enough data points to perform observation scaling"
+            " for keys: {}, need at least 2, have: {}".format(
+                events.keys, measured_data.shape[2]))
+
+    return measured_data
+
+
+def _filter_ensamble_std(ensamble_std, std_cutoff):
+    """
+    Filters on ensamble variation versus a user defined standard
+    deviation cutoff.
+    """
+    return ensamble_std < std_cutoff
+
+
+def _filter_ens_mean_obs(ensamble_mean, ensamble_std, observation_values, observation_std, alpha):
+    """
+    Filters on distance between the observed data and the ensamble mean based on variation and
+    a user defined alpha.
+    """
+    return abs(observation_values - ensamble_mean) > alpha * (ensamble_std + observation_std)
+
+
+def _combine_filters(filters):
+    df = pd.Series()
```

Comment:
In my opinion, descriptive variable names is quite important for the readability in Python.

Reply:
:+1:

Reply:
Changed to: 
```def _combine_filters(filters):
        combined_filter = pd.Series()
        for filter in filters:
            combined_filter = filter | combined_filter
        return combined_filter
```